src/server/database/sqlite.ts

```diff
@@ -96,6 +96,15 @@ async function initialSetup(db: DBServiceType) {
     });
   }
 
+  if (WG_INITIAL_ENV.ALLOWED_IPS) {
+    DB_DEBUG('Setting initial Allowed IPs...');
+    const userConfig = await db.userConfigs.get();
+    await db.userConfigs.update({
+      ...userConfig,
```

Comment:
adding the old user config doesn't add anything. you can also remove it in line 94, as this is a mistake